index.js

```diff
@@ -1,12 +1,15 @@
-global.Promise = require('bluebird')
-require('coffee-script/register')
 require('babel-core/register')
+require('ts-node').register()
+require('coffee-script/register')
```

Comment:
Am I correct in understanding that we’re _only_ AOT compiling client-side code? Do you have any insights into how much memory all of these are using at runtime once they’ve lazily loaded all the files?

Reply:
To clarify, I would really like us to AOT server-side code too.

Reply:
Yup, only client-side code, and will need to measure server-side memory usage. There is a doc that Cavvia and Joey and I have been working on [located here](https://docs.google.com/document/u/1/d/1818OAyc76N1jasRBuRNJFazNMXHHJId_rttF76RfG-k/edit) that we can add this line item to and investigate.